src/components/InlinePlayground.tsx

```diff
@@ -118,7 +118,7 @@ function InlinePlaygroundUI(props: { datastore: DataStore }) {
   );
 
   const handleChangeTab = (
-    event: React.ChangeEvent<{}>,
+    _event: React.ChangeEvent<object>,
```

Comment:
MUI's types here are kinda jank. This satisfies the type without using `{}`, since we can't use `HTMLElement` here.